e2e/test/scenarios/question/reproductions/38176-revert-to-previous-does-not-honor-parameters.cy.spec.js

```diff
@@ -0,0 +1,52 @@
+import { restore, questionInfoButton, rightSidebar } from "e2e/support/helpers";
+
+describe("issue 38176", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsNormalUser();
+    cy.intercept("PUT", "/api/card/**").as("updateQuestion");
+  });
+
+  it("restoring a question to a previous version should preserve the variables", () => {
+    cy.createNativeQuestion(
+      {
+        name: "30165",
+        native: {
+          query:
+            'SELECT "COUNTRY" from "ACCOUNTS" WHERE country = {{ country }} LIMIT 5',
+          "template-tags": {
+            country: {
+              type: "text",
+              id: "dd06cd10-596b-41d0-9d6e-94e98ceaf989",
+              name: "country",
+              "display-name": "Country",
+            },
+          },
+        },
+      },
+      { visitQuestion: true },
+    );
+
+    cy.findByPlaceholderText("Country").type("NL", { delay: 0 });
```

Comment:
Again, no need to shorten the delay on really short strings. Use this rarely, only when string is really long.